components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -321,6 +321,20 @@ ledger::InlineTipsPlatforms ConvertInlineTipStringToPlatform(
   return ledger::InlineTipsPlatforms::TWITTER;
 }
 
+bool ProcessPublisher(const GURL& url) {
+  // we should always process publisher on desktop
+  #if !defined(OS_ANDROID)
+    return true;
+  #endif
+
+  const std::string host = url.host();
+  if (host == "twitter.com") {
```

Comment:
that check should be replaced to `SameDomainOrHost` as for mobile it's `mobile.twitter.com` for example and that check will fail. That's how we do it for YouTube https://github.com/brave/brave-core/blob/master/browser/android/preferences/background_video_playback_tab_helper.cc#L42

Reply:
fixed